docker/fundamentals/networking/docker-port-forwarding.md

```diff
@@ -24,7 +24,7 @@ docker run \
   -t someimage
 ```
 
-Using the `EXPOSE` command in the Dockerfile, we can tell Docker that the container listens on the specified network port at runtime:
+Using the `EXPOSE` command in the Dockerfile, we can tell Docker that the container listens to the specified network port at runtime:
```

Comment:
Perhaps usually but in the realm of networking ["listening on a port"](https://www.google.com/search?sxsrf=ACYBGNSTARLLJLV2kzSbGwnGUdXXi_ONuw%3A1575637167351&ei=r1DqXaCHFYLt_QaArZ2IDQ&q=listening+on+port+&oq=listening+on+port+&gs_l=psy-ab.3..35i39j0l6j0i22i30l3.19083.19083..20421...0.2..0.81.81.1......0....1..gws-wiz.......0i71.HLvUx61jrRA&ved=0ahUKEwjgmp7tiaHmAhWCdt8KHYBWB9EQ4dUDCAs&uact=5) is quite normal.

Reply:
Ok cool - ignore the change then